front/src/applications/stdcm/components/SimulationReportSheet.tsx

```diff
@@ -55,6 +56,18 @@ const SimulationReportSheet = ({
     },
     [mapImageUrl]
   );
+  const getArrivalTime = (
+    step: PathStep,
+    { isFirstStep, isLastStep }: { isFirstStep: boolean; isLastStep: boolean }
```

Comment:
You can make isFirstStep and isLastStep nullable
```suggestion
    { isFirstStep, isLastStep }: { isFirstStep?: boolean; isLastStep?: boolean }
```

Reply:
fixed